### ISSUE TITLE
Make timeline accessible for keyboard users

### ISSUE THREAD

**Issue by Arnei** [elan-ev/opencast-editor#84]
The timeline and scrubber are inaccessible for users who cannot perform drag n' drop movements or have to rely on the keyboard. The ability to move the scrubber through keys, e.g. the arrow keys, may help in solving this problem

Solving issue #67 may also somewhat alleviate this problem.

**Comment by Arnei** [comment]
In a basic manner solved by #95.
